client/app/scripts/components/node-details/node-details-relatives-link.js

```diff
@@ -1,9 +1,10 @@
 import React from 'react';
+import { connect } from 'react-redux';
 
 import { clickRelative } from '../../actions/app-actions';
 import MatchedText from '../matched-text';
 
-export default class NodeDetailsRelativesLink extends React.PureComponent {
+class NodeDetailsRelativesLink extends React.Component {
```

Comment:
Because, as I understand it, Redux already does the `props` comparison the same way `PureComponent` does. In fact, `PureComponent` is a relatively new concept in Redux that we were simulating before by an *empty* Redux connect. :)